src/caustics/lenses/epl.py

```diff
@@ -196,6 +203,7 @@ def _r_omega(self, z, t, q):
         --------
         Tensor
             The value of `R * omega(phi)`.
+        *Unit: meters*
```

Comment:
The spacing doesn't match the rest of the usages.

Reply:
Updated